addons/web/static/src/legacy/js/core/ajax.js

```diff
@@ -167,8 +167,16 @@ var loadCSS = (function () {
             urlDefs[url] = new Promise(function (resolve, reject) {
                 $link.on('load', function () {
                     resolve();
-                }).on('error', function () {
-                    reject(new Error("Couldn't load css dependency: " + $link[0].href));
+                }).on('error', async () => {
+                    if (retryCount < 3) {
+                        await concurrency.delay(5000 + 2500 * retryCount);
+                        reject();
+                        delete urlDefs[url];
+                        $link.remove();
+                        loadCSS(url, retryCount + 1);
```

Comment:
Oh indeed, the Promise should not be blindly replaced - otherwise current awaits will not wait for a second chance. I should do the following:
- not `reject()` nor `delete`
- add a condition to the the first `if` to check if it is the first attempt
- chain the old `Promise` to the new one (if there was an old one)